cpp/include/cuml/cuml.hpp

```diff
@@ -98,11 +98,20 @@ class cumlHandle {
   int getNumInternalStreams();
 
   /**
-   * @brief API to get the internal streams as a vector.
-   * @return vector of internal streams in the handle
+   * @brief API to get the worker streams as a vector.
+   * @return vector of worker streams in the handle
    */
   std::vector<cudaStream_t> getInternalStreams() const;
```

Comment:
@jirikraus @cjnolet maybe it is now better to return a const-ref of the underlying vector, instead of creating a new vector everytime this method gets called? Any thoughts?

Reply:
I must admit I don't remember why and when `getInternalStreams()` was added. IIRC my initial proposal for this only had `cudaStream_t getInternalStream(int sid) const;`. However I fully agree if `getInternalStreams()` is needed it should **not** return `std::vector<cudaStream_t>` per value.